static/window-template.tmpl

```diff
@@ -0,0 +1,333 @@
+
+        <link rel="stylesheet" href="{{cookiecutter.cdn}}/styles.css"/>
+
+        <link rel="stylesheet" href="{{cookiecutter.cdn}}/xtermjsixel/xterm.css" />
+        <link rel="stylesheet" href="{{cookiecutter.cdn}}/xtermjsixel/style.css"/>
+        <style>
+            .hidden {
+                display: none;
+            }
+
+            #status {
+                display: inline-block;
+                vertical-align: top;
+                margin-top: 20px;
+                margin-left: 30px;
+                font-weight: bold;
+                color: rgb(120, 120, 120);
+            }
+
+            #progress {
+                height: 20px;
+                width: 300px;
+            }
+
+            div.emscripten { text-align: center; }
+            div.emscripten_border { border: 1px solid black; }
+            div.thick_border { border: 4px solid black; }
+
+            /* the canvas *must not* have any border or padding, or mouse coords will be wrong */
+            canvas.emscripten { border: 0px none; background-color: rgba(0, 0, 0, 0.4); }
+
+            body {
+                font-family: arial;
+                margin: 0;
+                padding: none;
+            }
+
+            /* average size of droid screen 470dp x 320dp  */
+            .topright{
+                position:absolute;
+                top: 8px;
+                right: 8px;
+            }
+
+            .topleft {
+                position:absolute;
+                top: 8px;
+                left: 8px;
+            }
+
+            .bottomright {
+                position:absolute;
+                bottom: 8px;
+                right: 8px;
+            }
+
+            .bottomleft {
+                position:absolute;
+                bottom: 8px;
+                left: 8px;
+            }
+
+            .center {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                margin: 0;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translate(-50% , -50%);
+            }
+
+            .full {
+                width: 100%
+                height: 100%
+            }
+
+            .trinfo{
+                position:absolute;
+                top:330px;
+                right:0px;
+                border: 1px solid black;
+            }
+
+            .framed{
+                position:absolute;
+                top:400px;
+                right:10px;
+                border: 1px solid black;
+            }
+        </style>
+
+        <script src="{{cookiecutter.cdn}}/xtermjsixel/xterm.js"></script>
+        <script src="{{cookiecutter.cdn}}/xtermjsixel/xterm-addon-image.js"></script>
+        <script src="{{cookiecutter.cdn}}/browserfs.min.js"></script>
+
+        <script type="module" src="{{cookiecutter.cdn}}/python311.js" async defer></script>
+
+
+    </head>
+
+    <body>
+        <div class="hidden">

```

Comment:
the hiding blocks shoud have all an "id" so python can reach them via the emscripten browser bridge which is compiled automatically with pygame.

and for that block specifically it should be named id="transfer" so emscripten can unhide it in case of serious js problem.

Reply:
i am thinking of removing those divs with the hidden tag and add the hidden element to those elements, so ig tht id won't be needed